src/odemis/driver/test/picoquant_test.py

```diff
@@ -168,5 +168,172 @@ def _on_rawdet(self, df, data):
         self._lastdata = data
 
 
+PH300_KWARGS = dict(name="Time Correlator", role='time-correlator', device="fake",
+                    shutter_axes={"detector0": ["shutter0", "x", 0, 1], "detector1": ["shutter1", "x", 0, 1]})
+TC_ACT_KWARGS = dict(name="Time-Correlator Actuators", role=None, port="/dev/ttyACM0",
+                     address=None, axes=["fw", "od"], ustepsize=[3.392e-5, 3.272e-5],
+                     unit=["rad", "rad"], refproc="Standard", refswitch={"fw": 0, "od": 0},
+                     do_axes={4: ["shutter0", 0, 1, 1], 5: ["shutter1", 0, 1, 1]},
+                     led_prot_do={4: 0, 5: 0})
+
+if TEST_NOHW:
+    PH300_KWARGS["device"] = "fake"
+    TC_ACT_KWARGS["port"] = "/dev/fake6"
+
+
+class TestPH300_Shutters(unittest.TestCase):
```

Comment:
It seems you've broken `TestPH300`, as it has no detectors now.
BTW, `TestPH300_Shutters` could probably inherit from `TestPH300` and become a lot slimmer.